lib/consul/async/process_handler.rb

```diff
@@ -9,6 +9,14 @@ class ProcessDoesNotExist < StandardError
     class ProcessHandler
       attr_reader :command, :sig_reload, :sig_term, :pid, :exit_status, :last_signal_sent, :reload_scheduled
       attr_writer :reload_scheduled
+
+      def restart
+        warn "Restart process with pid #{pid}"
+        kill
```

Comment:
Why not using Process.kill("XXXX", pid)?

I suppose the pid will be changed

Reply:
yes, it's why I use the kill procedure ! it's do the Process.kill("XXXX", pid)? for me

Reply:
ok my bad - just saw it - ok then